conans/test/functional/generators/cmake_generator_test.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+
+from nose.plugins.attrib import attr
+
+from conans import tools
+from conans.test.utils.tools import TestClient
+
+
+CONAN_RECIPE = """
+from conans import ConanFile, CMake
+
+class FooConan(ConanFile):
+    name = "foo"
+    version = "0.1"
+    settings = "os_build", "compiler", "build_type"
+    generators = "cmake"
+    exports = '*'
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+"""
+
+
+CPP_CONTENT = """
+int main() {}
+"""
+
+CMAKE_RECIPE = """
+cmake_minimum_required(VERSION 2.8.12)
+project(dummy CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(dummy dummy.cpp)
+"""
+
+
+@attr("slow")
+class CMakeGeneratorTest(unittest.TestCase):
+
+    def _generator_test_helper(self, os_build):
```

Comment:
This test could be simplified a bit:
- No actual compiling is necessary. You can remove the dummy.cpp, and even the generation of ``conanbuildinfo.cmake``
- No need to create, can remove name, version, and do a ``conan install`` + ``conan build``, which would be slightly faster.

Reply:
But we can check for "MinGW Makefiles" and "Unix Makefiles" in the output when building the package.